br/pkg/lightning/backend/local/local.go

```diff
@@ -1133,6 +1133,22 @@ const (
 	retryIngest
 )
 
+<<<<<<< HEAD
```

Comment:
conflict not resolved ?

Reply:
some conflict comes from https://github.com/pingcap/tidb/pull/36874, will fix it after that pr merged

Reply:
fixed, ptal